cli/compose/loader/loader_test.go

```diff
@@ -332,6 +332,8 @@ func TestInvalidTopLevelObjectType(t *testing.T) {
 }
 
 func TestNonStringKeys(t *testing.T) {
+	// FIXME(thaJeztah): opkg.in/yaml.v3, which always unmarshals to a map[string]any, so we cannot produce a customized error for invalid types.
+	t.Skip("not supported by gopkg.in/yaml.v3, which always unmarshals to a map[string]any")
 	_, err := loadYAML(`
```

Comment:
So, I kept the test for our custom  errors (this test runs with badly formed YAML files to test the custom  errors).

But mostly as a "reminder" in case someone would like to bite their teeth into implementing something like that.

(but in all honesty, we can probably just remove the test and call it a day; just write proper YAML)